index.html

```diff
@@ -2,316 +2,349 @@
+            </section>
+        </section>
+        <div class="languages_for_web">
+            <section class="languages">
+                <a href="#" class="languages-heading">
+                    <h3 class="font-style">Languages</h3>
+                    <img class="vector" src="images/arrow-down.png" alt="arrow down" />
+                </a>
+                <ul class="lang">
+                    <li>
+                        <a class="language" href="#lang">
+                            <img src="images/js.svg" alt="javascript logo" />
+                            <h4 class="language-text javascript font-style">
+                                JavaScript
+                            </h4>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="language m-left" href="#lang">
+                            <img src="images/html.svg" alt="html logo" />
+                            <h4 class="language-text html font-style">HTML</h4>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="language m-left" href="#lang">
+                            <img src="images/css.svg" alt="css logo" />
+                            <h4 class="language-text html font-style">CSS</h4>
+                        </a>
+                    </li>
+                </ul>
+            </section>
+            <section class="frameworks">
+                <div class="framework-list">
+                    <h3 class="font-style">Frameworks</h3>
+                    <img class="vector" src="images/right-arrow.png" alt="arrow right" />
+                </div>
+                <div class="framework-list">
+                    <h3 class="font-style">Skills</h3>
+                    <img class="vector" src="images/right-arrow.png" alt="arrow right" />
+                </div>
+            </section>
+        </div>
+
+
+    </div>
+
+    <footer id="contact" class="footer">
+        <div class="contact-wrapper">
+            <h2 class="contact font-style">Contact me</h2>
+            <p class="contact-description font-style">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d love
+                to help with it
+            </p>
+            <form id="contact-form" action="https://formspree.io/f/xrgjnypg" method="post" class="form-wrapper">
+                <fieldset class="form-fieldset">
+                    <div class="form-group">
+                        <input type="text" name="name" id="name" class="form-input font-style" placeholder="Your name"
+                            required maxlength="30" data-field="name" />
+                    </div>
+                    <div class="form-group">
+                        <input type="email" name="email" id="email" class="form-input font-style"

```

Comment:
- [ ] I would kindly suggest you to consider to remove the footer out of the form sections ,you are using a footer tag instead of a sections and all form inpute should be wrap in one `form tag` for a semantic reason and easy to read.